pkg/cmd/target/control-plane.go

```diff
@@ -36,5 +38,13 @@ gardenctl target control-plane --garden my-garden --project my-project --shoot m
 
 	o.AddFlags(cmd.Flags())
 
+	manager, err := f.Manager()
+	utilruntime.Must(err)
```

Comment:
It is not acceptable to assert no erros here. We never have done this anywhere else.